simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/StationSimulatorRunner.java

```diff
@@ -50,6 +50,17 @@ public void run() {
         });
     }
 
+    /**
+     * Stops OCPP-compliant stations.
+     */
+    public void stop() {
```

Comment:
looks like stopAll

Reply:
i named it stop() (instead of stopAll) because there is an equivalent method called run(), that connects to all specified stations.